src/devtools/devtools.tsx

```diff
@@ -316,7 +336,7 @@ export function ReactQueryDevtools({
                   bottom: '0',
                   left: '0',
                 }),
-            ...toggleButtonStyle,
+            ...(toggleButtonStyle as any),
```

Comment:
ah, there's another one:

```suggestion
            ...(toggleButtonStyle as unknown),
```

Reply:
@TkDodo  it looks like there might be some package that is using an incompatible version of @types/react, Fixed it with [yarn resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) for now as recommended by this comment - https://github.com/react-dropzone/react-dropzone/issues/1000#issuecomment-880533066